content/references/protocol-reference/transactions/transaction-types/xchainclaim.md

```diff
@@ -0,0 +1,65 @@
+---
+html: xchainclaim.html 
+parent: transaction-types.html
+blurb: Create a bridge between two chains.
+labels:
+  - Interoperability
```

Comment:
If you want this to link to a landing page that actually shows all the pages with this label, you need to add a stanza for this landing page to `dactyl-config.yml`. Here's an example: https://github.com/XRPLF/xrpl-dev-portal/blob/abd57c9db602c829846f3575abf4bb44c708acd2/dactyl-config.yml#L4058-L4067

You should add the page to the `- ja` target with a `# TODO: translate title and blurb` comment to remind us to come back and translate it, too.